tests/unit/models/test_sbom.py

```diff
@@ -141,3 +272,252 @@ def test_sort_and_dedupe_components(self) -> None:
+                            "referenceLocator": "pkg:golang/github.com/org/A@v1.1.0?repository_id=R1",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "github.com/org/A",
+                    "version": "v1.1.0",
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/github.com/org/A@v1.1.0?repository_id=R2",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "github.com/org/A",
+                    "version": "v1.0.0",
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/github.com/org/A@v1.0.0",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "github.com/org/A",
+                    "version": "v1.0.0",
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/github.com/org/A@v1.0.0",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "github.com/org/B",
+                    "version": "v1.0.0",
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/github.com/org/B@v1.0.0",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "fmt",
+                    "version": None,
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/fmt",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+                {
+                    "name": "bytes",
+                    "version": None,
+                    "externalRefs": [
+                        {
+                            "referenceCategory": "PACKAGE-MANAGER",
+                            "referenceLocator": "pkg:golang/bytes",
+                            "referenceType": "purl",
+                        }
+                    ],
+                },
+            ],
+        )
+        print(sbom.packages)
```

Comment:
extra print

Reply:
It's the same as in existing test where is also print. I thought it's on purpose.

Reply:
it's probably just forgotten there